frame/treasury/src/benchmarking.rs

```diff
@@ -217,3 +217,25 @@ benchmarks! {
 		Treasury::<T>::on_initialize(T::BlockNumber::zero());
 	}
 }
+
+#[cfg(test)]
+mod tests {
+	use super::*;
+	use crate::tests::{new_test_ext, Test};
+	use frame_support::assert_ok;
+
+	#[test]
+	fn test_benchmarks() {
+		new_test_ext().execute_with(|| {
+			assert_ok!(test_benchmark_propose_spend::<Test>());
+			assert_ok!(test_benchmark_reject_proposal::<Test>());
+			assert_ok!(test_benchmark_approve_proposal::<Test>());
+			assert_ok!(test_benchmark_report_awesome::<Test>());
+			assert_ok!(test_benchmark_retract_tip::<Test>());
+			// assert_ok!(test_benchmark_tip_new::<Test>());
+			// assert_ok!(test_benchmark_tip::<Test>());
+			// assert_ok!(test_benchmark_close_tip::<Test>());
```

Comment:
There is an issue where I am unable to modify the fixed TenToFourteen Contains struct.

This is needed to test these benchmarks, and I could use help on how to do that,

Reply:
I used the refcell pattern here to create a mutable list of members which I then referenced in the `TenToFourteen` struct to get me the right information.